osdep/terminal-unix.c

```diff
@@ -492,8 +492,8 @@ void terminal_setup_getch(struct input_ctx *ictx)
     }
 
     setsigaction(SIGINT,  quit_request_sighandler, SA_RESETHAND, false);
-    setsigaction(SIGQUIT, quit_request_sighandler, SA_RESETHAND, false);
-    setsigaction(SIGTERM, quit_request_sighandler, SA_RESETHAND, false);
+    setsigaction(SIGQUIT, quit_request_sighandler, 0, false);
+    setsigaction(SIGTERM, quit_request_sighandler, 0, false);
```

Comment:
One more thing, it doesn't make much sense to not mask signals now that we don't use `SA_RESETHAND` anymore. Change those trailing `false` argument to `true`.

Reply:
Thanks